helix-rest/src/main/java/org/apache/helix/rest/clusterMaintenanceService/StoppableInstancesSelector.java

```diff
@@ -258,18 +293,21 @@ private Map<String, Set<String>> getOrderedZoneToInstancesMap(
   }
 
   /**
-   * Collect instances marked for evacuation in the current topology and add them into the given set
+   * Collect instances marked for evacuation, swap, or unkown in the current topology and add
```

Comment:
nit: I would suggest refine on comment here. I think we are referring to all the instances in the cluster, not necessarily the topology aware cluster's topology tree.